cmd/catalog/main.go

```diff
@@ -87,6 +93,10 @@ func main() {
 		log.Panicf("error configuring operator: %s", err.Error())
 	}
 
+	http.Handle("/metrics", promhttp.Handler())
+	go http.ListenAndServe(":8081", nil)
```

Comment:
This is serving a nil handler - did you mean to serve metrics on this port?

Reply:
This port change was intentional, though perhaps it's not the best? OLM serves healthz from port 8080, so I wanted the metrics ports to be consistent.